code/workspaces/infrastructure-api/src/controllers/volumeController.spec.js

```diff
@@ -89,6 +89,13 @@ describe('Volume Controller', () => {
       await executeCreateValidator(requestBody);
       expectValidationError('volumeSize', 'Volume Size must be an integer between 5 and 200');
     });
+
+    it('should validate the type is between 1 and 2', async () => {
+      const requestBody = createRequestBody();
+      requestBody.type = 0;
+      await executeCreateValidator(requestBody);
+      expectValidationError('type', 'Type muste be one of glusterfs,nfs');
```

Comment:
typo in message: Type _must_ be one of...

Reply:
Thanks, corrected.